### PR TITLE
Initial implementation of SKU Selector

### PR DESCRIPTION
**Pull request by gugabribeiro**
#### What is the purpose of this pull request?
Add the initial implementation of SKU Selector component.

#### What problem is this solving?
New feature/component.

#### Screenshots or example usage
<a href="https://gustavo--storecomponents.myvtex.com/sku-selector">Workspace</a>

captura de tela de 2018-05-22 11-29-43
captura de tela de 2018-05-22 11-29-59
captura de tela de 2018-05-22 11-30-49

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/SKUSelector/components/Selector.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          { this.props.title }
+        </div>
+        <div className="inline-flex flex-wrap"> 
+          {
+            React.Children.map(this.props.children, (child, index) => {
+              if (child.type !== SelectorItem) {
+                return child
+              } else {
+                return React.cloneElement(child, {
+                  index: index,
```
Comment (ClaudivanFilho) [inline react/components/SKUSelector/components/Selector.js]: `index, index` -> `index`

File: react/components/SKUSelector/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# Product details
+
+SKU Selector component.
```
Comment (brunojdo) [inline react/components/SKUSelector/README.md]: This is our ...

File: react/components/SKUSelector/README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+And to import it in your code: 
+
+```js
+import SKUSelector from 'vtex.storecomponents/SKUSelector'
+```
+
+## Usage
+
+You can use in your code like a React component with the jsx tag: `<SKUSelector />`.
```
Comment (brunojdo) [inline react/components/SKUSelector/README.md]: use it into...

File: react/components/SKUSelector/README.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
+# Product details
+
+SKU Selector component.
+
+## Installation
+
+To install this component, you have two ways. Follow the instructions bellow.
```
Comment (brunojdo) [inline react/components/SKUSelector/README.md]: below

File: react/components/SKUSelector/README.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+        categories: [
+          { name: 'Red' },
+          { name: 'Blue' }
+        ]
+      ]
+    }
+  ]}
+/>
+```
+## Props
```
Comment (brunojdo) [inline react/components/SKUSelector/README.md]: `###`

**Pull request merged**